controller/user-controller.js

```diff
@@ -0,0 +1,16 @@
+const userService = require('../services/user-services');
+
+const signUp = async (req, res) => {
+  try {
+    const {email, password, firstname, lastname} =req.body;
+
+    await userService.signUp(email, password, firstname, lastname);
+
+    res.status(201).end();
```

Comment:
회원가입이 완료되었음을 클리이언트 (프론트엔드 개발자)에게 알리는 메시지를 전달하도록 수정해주세요.

```suggestion
    return res.status(201).send({ message : "SIGNUP_SUCCESS" });
```